app.py

```diff
@@ -1,11 +1,22 @@
 import json
+import os
 from flask import Flask, request, jsonify
 from flask.views import MethodView
 from werkzeug.exceptions import HTTPException
 from error_handling import CustomError, ValidationError, WrongIdError
 
 app = Flask(__name__)
-with open("restaurants.json", "r") as read_file:
+try:
+    env = os.environ['FLASK_ENV']
```

Comment:
1. what will happen if incorrect `FLASK_ENV` is given? e.g. I set `FLASK_ENV` to `lol` such a case should be handled
2. FYI, there is a way to eliminate `try...catch` block like that:
   ```python
   os.environ.get('FLASK_ENV', 'development')`
   ```
   it will use `development` if another is not defined

Another thing to consider: you are building the software for production. While during development you are in the development environment, you should always think about production first.
That means that if `FLASK_ENV` is not defined, then it should be `production` by default

Reply:
fixed